packages/web/localizations/en.json

```diff
@@ -830,7 +827,8 @@
     "tvl": "TVL",
     "backButton": "All tokens",
     "ariaBackButton": "All tokens back button",
-    "noData": "N.D."
+    "noData": "N.D.",
+    "buyToken": "Buy {coinDenom}"
```

Comment:
Looks like these still need to be translated?

Reply:
Yes, we're still on draft, that's why a lot of things are still missing

Reply:
Fixed